docs/sources/variables/chained-variables.md

```diff
@@ -0,0 +1,174 @@
++++
+title = "Chained variables"
+keywords = ["grafana", "templating", "variable", "nested", "chained", "linked"]
+type = "docs"
+[menu.docs]
+weight = 600
```

Comment:
Shouldn't be needed any longer since we have the meny.yaml
```suggestion
```

Reply:
We want to get rid of menu.yaml and go back to the standard Hugo method of creating the side menu, which is based on folder structure and weight.

Reply:
Ohh great 👍